ingress/controllers/gce/BETA_LIMITATIONS.md

```diff
@@ -120,3 +122,42 @@ gcloud container clusters create mycluster --network "default" --num-nodes 1 \
 --disk-size 50 --scopes storage-full
 ```
 
+## Changing the cluster UID
+
+The Ingress controller configures itself to add the UID it stores in a config map in the `kube-system` namespace.
+
+```console
+$ kubectl --namespace=kube-system get configmaps
+NAME          DATA      AGE
+ingress-uid   1         12d
+
+$ kubectl --namespace=kube-system get configmaps -o yaml
+apiVersion: v1
+items:
+- apiVersion: v1
+  data:
+    uid: UID
+  kind: ConfigMap
+...
+```
+
+You can pick a different UID, but this requires you to:
+
+1. Delete existing Ingresses
+2. Edit the configmap using `kubectl edit`
+3. Recreate the same Ingress
+
+After step 3 the Ingress should come up using the new UID as the suffix of all cloud resources. You can't simply change the UID if you have existing Ingresses, because
+renaming a cloud resource requires a delete/create cycle that the Ingress controller does not currently automate. Note that the UID in step 1 might be the empty string,
+if you had a working Ingress before upgrading to Kubernetes 1.3.
+
```

Comment:
Do these instructions apply to everyone with ingresses configured and upgrading to 1.3 ? Is the UID going to change from "" to something non-empty when we upgrade to 1.3 ?